### ISSUE TITLE
[Feature request]  define a range of characters to exclude in single setting

### ISSUE THREAD

**Issue by sheldonhull** [nhoizey/vscode-gremlins#184]
**Is your feature request related to a problem? Please describe.**

I'd like to exclude the entire range of characters in a given block. 
Say for example the [Latin Supplement 1](https://unicode-table.com/en/blocks/latin-1-supplement/)

I've had accidental input catch me and cause a lot debugging effort after accidentally including some special characters from special characters on the mac keyboard.

**Describe the solution you'd like**

An example below from the unicode range mentioned from the link above.
I'd like to flag all extended nonstandard code to catch them.

```json
        "0080-00FF": {
            "level": "error",
            "zeroWidth": false,
            "description": "Invalid Character Latin-1 Supplement",
            "overviewRulerColor": "rgba(255,127,80,1)",
        },
```

**Describe alternatives you've considered**
Also trying to figure out how to disable in the keyboard settings for system, but in the meantime catching these in code would be fantastic. Otherwise I need to script a large amount of unicode characters in json to be able to flag it properly which is pretty clunky.

edit: for those that are newer to macOS like me... you can fix the option modifier by going to: `System Preferences > Keyboard > Modifier Keys > Option Key > No Actions`. Still want to improve gremlins to catch these, but now at least I won't accidentally throw those in my code :grin:

**Comment by TheSench** [comment]
@sheldonhull This should actually already be implemented exactly as you have it stated.  Have you given it a try?

**Comment by sheldonhull** [comment]
@TheSench yes, I tried many different ways and didn't seem to pick up the characters. Are you saying the way I demonstrated above with the dash in the middle should correctly work as is? I'll test again with this if so.

**Comment by sheldonhull** [comment]
@TheSench hmm, maybe when I tried before it had a smart dash in the middle or something. I commented out all the specific rules for this I created and then tried:

```
    "0080-00FF": {
            "level": "error",
            "zeroWidth": false,
            "description": "Invalid Character Latin-1 Supplement",
            "overviewRulerColor": "rgba(255,127,80,1)",
        },
```

Tried the following characters:
»
×
Ö

![image](https://user-images.githubusercontent.com/3526320/118860330-e0291900-b8a0-11eb-96be-a0a0d9c94d40.png)

I think it works now. Thank you! 💯
